src/node_native_module.cc

```diff
@@ -36,6 +36,78 @@ using v8::String;
 using v8::Uint8Array;
 using v8::Value;
 
+void NativeModuleLoader::InitializeModuleCategories() {
+  if (module_categories_.is_initialized) {
+    DCHECK(!module_categories_.can_be_required.empty());
+    return;
+  }
+
+  std::set<std::string> prefixes = {
```

Comment:
Maybe `std::vector` or `std::array`, since this is only locally used and we don’t need the extra structure of a set?